src/prefect/client/client.py

```diff
@@ -727,7 +727,7 @@ def _request(
             headers.update(self._attached_headers)
 
         session = requests.Session()
-        retry_total = 6 if prefect.config.backend == "cloud" else 1
+        retry_total = 12
```

Comment:
Hi! I think I'm okay with removing the conditional check here but is there a particular reason you bumped this to 12? With the exponential backoff, the last sleep would be ~34 minutes. Can you move back to 6?

Reply:
Sure